libs/data-mapper/src/lib/components/mapOverview/MapOverview.tsx

```diff
@@ -77,9 +77,13 @@ const OverviewReactFlowWrapper = () => {
     fitView(reactFlowFitViewOptions);
   }, [fitView, sourceSchema?.schemaTreeRoot.key, targetSchema?.schemaTreeRoot.key]);
 
+  const schemaNodeType = {
```

Comment:
Move this outside the FC or memoize it where it is so ReactFlow doesn't get mad at you

Reply:
*IIRC

Reply:
I believe you are correct. Changed